core/contracts/oracle/implementation/Governor.sol

```diff
@@ -17,11 +17,13 @@ import "@openzeppelin/contracts/math/SafeMath.sol";
 contract Governor is MultiRole, Testable {
     using SafeMath for uint;
 
+    /****************************************
```

Comment:
FWIW: I really like these star boxes, they organize the contracts intuitively and will make reading the code in the future much easier